### PR TITLE
feat: sound manager added

### PR DESCRIPTION
**Pull request by saledjenic**

### CODE REVIEW

File: lib/src/Status/SoundManager.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+void SoundManager::playSound(const QUrl &soundUrl)
+{
+	if (m_player->state() != QMediaPlayer::PlayingState)
+	{
+        if (m_player->currentMedia().canonicalUrl() != soundUrl)
+        {
+            m_player->setMedia(soundUrl);
+        }
```
Comment (jrainville) [inline lib/src/Status/SoundManager.cpp:23]: The indentation looks messed up here

**Review by jrainville — APPROVED** [review]
Looks good.
Just take a look at the indentation if it's not just a diff bug

**Review by jrainville — APPROVED** [review]

File: lib/include/DOtherSide/Status/SoundManager.h [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+         */
+        SoundManager(SoundManager &&) = delete;
+        /*!
+         * Delete copy asignment operator.
+         */
+        SoundManager &operator=(const SoundManager &) = delete;
+        /*!
+         * Delete move asignment operator.
+         */
+        SoundManager &operator=(SoundManager &&) = delete;
```
Comment (borismelnik) [inline lib/include/DOtherSide/Status/SoundManager.h:35]: You can use just `Q_DISABLE_COPY_MOVE` instead of methods above

**Review by borismelnik — CHANGES_REQUESTED** [review]
Nice work! 
But you know, the best way for playing raw audio is `QSound`(without any instance) or `QSoundEffect` (for a low latency file). 
`QMediaPlayer` needed for playing songs and etc.

**Review by borismelnik — APPROVED** [review]

**Review by jrainville — APPROVED** [review]

**Pull request merged**